README.md

```diff
@@ -22,6 +22,23 @@ TorchDriveEnv comes bundled with a predefined collection of diverse and challeng
 |------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Three Way | Chicken | Parked Car | Roundabout | Traffic Lights |
 
+## Custom Scenarios
+If you would like to define your own scenarios, Inverted AI provides a tool [here](https://www.inverted.ai/portal/scenario-builder) for creating custom scenarios. 
+
+### Creating Scenarios
+* To create a scenario, first select a location from the list of locations provided from the scenario builder.
+
+* Choose an area of interest by using the `Rendering FOV` slider to adjust the zoom, and use `Set Map Center` to center the map on the desired area of interest.
+
+* (*Optional*) Add cars manually with `Add Car to Scenario`, or adjust the sliders to the number of cars needed and press `Initialize New Scenario` to let INITIALIZE do the work for you. Edit the cars as needed by dragging, rotating, and setting their initial speed conditions.
+    * Cars in the scenario can be marked as stationary vehicles by selecting them on the map, then selecting `Mark as Parked`.
+    * If you would like a car to follow a certain trajectory, first select the car, then select `Add Waypoints` to define a waypoint sequence for the car to follow. To end a sequence, click on the last set waypoint.
+    * If you would like to validate your waypoints and initial conditions, you can click on `DRIVE` to run an example of what your simulation might look like, and adjust the scenario as needed.
+
+* Define your ego vehicle initialization range and waypoints with `Start Ego Waypoint Sequence`. The first two points define the range where the starting position is sampled from. To end a sequence, click on the last set waypoint.
+
+* Export your scenario JSON files to the same directory and use [load_labeled_data(data_dir)](https://github.com/inverted-ai/torchdriveenv/blob/master/torchdriveenv/env_utils.py#L31) to load the scenarios.
```

Comment:
Is there any way to make a script that would take a path to the JSON file, construct the environment, and display something that shows that the scenario was loaded correctly? It's fine to use some dummy policy like a zero action.

Reply:
![image](https://github.com/inverted-ai/torchdriveenv/assets/26733898/f2e98e8c-f48d-43aa-895c-bbf8aac6b21d)
You could load a JSON file via the labeling tool.

Reply:
Yes, but I'd like to construct an environment and see that it's playing the scenario. That would provide a much easier entry point than having to figure out where the `load_labeled_data` function needs to be called and how to see that it's executing the right scenario.

Reply:
I see. You could try: https://github.com/inverted-ai/torchdriveenv/blob/master/examples/waypoint_suite_env_example.ipynb, replace the `load_default_validation_data()` with `load_labeled_data`. That's how I generate the video in this PR.

Reply:
This notebook doesn't run. It seems that `load_default_env_config` no longer exists.